metatron/src/main/java/uk/gov/ida/eidas/metatron/domain/EidasConfig.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.ida.eidas.metatron.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
+
+public class EidasConfig {
+    @JsonProperty
+    private List<EidasCountryConfig> countries = new ArrayList<>();
+
+    @JsonIgnore
+    private Map<String, EidasCountryConfig> countriesMap;
+
+    public Collection<EidasCountryConfig> getCountries() {
+        return countries;
+    }
+
+    public EidasCountryConfig getCountry(String country) {
+        if(Objects.isNull(countriesMap)) {
+            countriesMap = new HashMap<>();
+            countries.stream().forEach(eidasCountryConfig -> countriesMap.put(eidasCountryConfig.getName(), eidasCountryConfig));
+        }
+        return countriesMap.get(country);
+    }
+
+    public void retainAll(Function<EidasConfig, Collection<EidasCountryConfig>> filterFunction) {
```

Comment:
I can't see this method being used anywhere. Is it needed? What does it do?

Reply:
This is a hangover from the config repo where this class was born, this and the other method can go as they're not used here.